sample/SampleAspNetCoreApp/Controllers/HomeController.cs

```diff
@@ -119,6 +119,13 @@ public IActionResult Sample(int id)
 			return Ok(id);
 		}
 
+		public IActionResult SampleWithCustomName()

```

Comment:
It is little bit confusing what “sample” means here - I would go with `TransactionWithCustomName`

Reply:
Done